.github/CODEOWNERS

```
@@ -162,8 +162,10 @@ packages/react-tooltip @microsoft/cxe-red @behowell @khmakoto
 packages/react-list @microsoft/teams-prg
 packages/react-toolbar @microsoft/teams-prg @chpalac @ling1726
 packages/react-portal-compat @microsoft/teams-prg
+#packages/react-alert @microsoft/teams-prg
```

```suggestion
packages/react-alert @microsoft/teams-prg
```

Was it commented out by the autogeneration ?

---

Yes, looks like it. I didn't make any edits to any files.